crates/turbo-static/src/call_resolver.rs

```diff
@@ -0,0 +1,125 @@
+use fjall::PartitionCreateOptions;
+
+use crate::{lsp_client::RAClient, Identifier, IdentifierReference};
+
+pub struct CallResolver<'a> {
+    client: &'a mut RAClient,
+    fjall: &'a fjall::Keyspace,
```

Comment:
I'm worried that a library like `fjall` might be overkill for our needs, especially as it looks like you don't need many of the things if offers (range and prefix search, snapshotting, etc). How large is this dataset? Would it be practical to just keep it in memory as a hashmap, and then serialize to a binary format (e.g. with rkyv or serde) when writing it to disk?

Reply:
I have replaced it. It absolutely is overkill so it has been replaced with a simple hashmap. I wanted to try it out in case it made sense for persistent caching down the line so it was really just a chance to see what it performs like and learn the library a little.